code-samples/eventing/bookstore-sample-app/db/sample.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS book_reviews(
+
+  id SERIAL PRIMARY KEY,
+  post_time datetime NOT NULL,

```

Comment:
Postgres doesn't support datetime as a type, it uses timestamp instead
```suggestion
  post_time timestamp NOT NULL,
```

Reply:
Thank you so much for the reviews!